source/iostreams.tex

```diff
@@ -12311,8 +12311,9 @@
 \end{itemize}
 
 \pnum
-If the encoding being converted to has no representation for source
-characters, the resulting converted characters, if any, are unspecified.
+If the encoding being converted to has no representation for
+any of the characters in the argument or return value,
```

Comment:
This seems ambiguous. No representation "for any of the characters" could mean that every character has no representation, or at least one of the characters has no representation. The latter is intended.

Maybe something like "if any of the characters in the argument or return value has no representation in the destination encoding", or just replace "any" with "one or more".

Reply:
done